common/include/common/common_utils.hpp

```diff
@@ -717,17 +682,12 @@ static inline std::vector<syrk_param_t<scalar_t>> get_syrk_params(Args& args) {
   if (args.csv_param.empty()) {
     warning_no_csv();
     std::vector<syrk_param_t<scalar_t>> syrk_default;
-    constexpr index_t dmin = 64, dmax = 1024;
+    constexpr index_t dmin = 2048, dmax = 32768;
     constexpr scalar_t alpha{1};
     for (char uplo : {'u', 'l'}) {
       for (char trans : {'n', 't'}) {
-        for (index_t n = dmin; n <= dmax; n *= 2) {
-          for (index_t k = dmin; k <= dmax; k *= 2) {
-            for (scalar_t beta : {0, 1}) {
-              syrk_default.push_back(
-                  std::make_tuple(uplo, trans, n, k, alpha, beta));
-            }
-          }
+        for (index_t n = dmin; n <= dmax; n *= 4) {
+          syrk_default.push_back(std::make_tuple(uplo, trans, n, n, alpha, 0));
```

Comment:
Thank you for you patch @aacostadiaz.
For this benchmark's parameter I see we are not testing for `beta != 0`. I think this may reduce the number of operations carried out by the routine, I suggest you to set `beta` to 1 as it is for `alpha`